react/src/components/molecules/HeaderSearch/index.js

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { componentWithName } from 'airbnb-prop-types';
```

Comment:
nice! we should slowing replacing the `componentPropTypeCheck` with `componentWithName`. Do they have a proptype with multiple components?

Reply:
They do!

Reply:
cool! we should convert the other proptype checks dependent on the custom functions and convert them to use airbnb-prop-types